controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -137,11 +129,14 @@ func (r *StorageClassClaimReconciler) Reconcile(ctx context.Context, request rec
 func (r *StorageClassClaimReconciler) reconcileConsumerPhases() (reconcile.Result, error) {
 	r.log.Info("Running StorageClassClaim controller in Consumer Mode")
 
-	providerClient, err := providerclient.NewProviderClient(r.ctx, r.storageCluster.Spec.ExternalStorage.StorageProviderEndpoint, time.Minute) // TODO what should be the timeout limit for synchronous operations on the client?
+	providerClient, err := providerclient.NewProviderClient(r.ctx, r.storageCluster.Spec.ExternalStorage.StorageProviderEndpoint, 15*time.Minute) // TODO Timeout based on StorageCluster's timeout for IBMCloud or IBMCloudCos.
```

Comment:
@nb-ohad I wasn't able to find any suitable timeouts as per the one you mentioned. Could you please point me to the file where that limit is used (so I can substitute that here)?

Reply:
Fixed: https://github.com/red-hat-storage/ocs-operator/pull/1608/files#diff-b60803ce0dfb3bc9a7a20db01eec72ffd2803fc90febca9acd9a0cea7af477dfR132. Thanks.